conf/configuration.go

```diff
@@ -29,60 +30,79 @@ type (
 	// LogLevel defines the logging level of the entire service.
 	// ServerName holds the name of the current server. This name will be used
 	// 	for identifying which servers are pinning a given skylink.
+	// SiaAPIPassword is the apipassword for the local skyd
+	// SiaAPIHost is the hostname/IP of the local skyd
+	// SiaAPIPort is the port of the local skyd
```

Comment:
why are these descriptions not in-line with the struct like we normally do, now I have to scan the docstring for an explanation of `SiaAPIPort` which is 10 lines up and I have to also know to consult the docstring for that info 😛  I sound like the most anal person ever saying that but I don't see the benefit of this new way of documenting struct fields

Reply:
having comments on each field of a struct makes it very hard for me to read, so I moved those to the overall description of the type. reverted, the comments are now per-field.